### PR TITLE
Make event classes final

### PR DESCRIPTION
**Pull request by darthf1**
As mentioned in https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1734#discussion_r532887585

### CODE REVIEW

File: src/Event/PostIndexPopulateEvent.php [hunk 1]
```diff
@@ -11,6 +11,6 @@
 
 namespace FOS\ElasticaBundle\Event;
 
-class PostIndexPopulateEvent extends IndexPopulateEvent
+final class PostIndexPopulateEvent extends IndexPopulateEvent
```
Comment (deguif) [inline src/Event/PostIndexPopulateEvent.php:14]: Maybe we should also mark the corresponding base class as `abstract`?

Comment (darthf1) [reply src/Event/PostIndexPopulateEvent.php:14]: done

Comment (darthf1) [reply src/Event/PostIndexPopulateEvent.php:14]: Hm tests fail if I add abstract. Should the tests be changed or should we refrain from adding abstract? 

https://travis-ci.com/github/FriendsOfSymfony/FOSElasticaBundle/jobs/451397287

Comment (XWB) [reply src/Event/PostIndexPopulateEvent.php:14]: An abstract class should not be initialized.

If the class was not meant to be abstract, removing the keyword should be enough.

Comment (darthf1) [reply src/Event/PostIndexPopulateEvent.php:14]: Ok, then its done. The result is no abstract classes.

**Pull request merged**